docker/php/Dockerfile

```diff
@@ -72,6 +72,6 @@ COPY docker/php/start.sh /usr/local/bin/docker-app-start
 RUN chmod +x /usr/local/bin/docker-app-start
 
 # Needed to fix permissions error
-USER 1000
+USER 100
```

Comment:
Doesn't this break standard `docker-compose`?

Reply:
Looks like `docker-compose` still works. 
I don't know why but it works 🤔

Reply:
I tested it work on my side.

The main issue I had with `docker-sync` is permission issues, it does not seem to work with an user ID of `1000` (I don't know why)